pandas/tests/frame/methods/test_drop.py

```diff
@@ -268,6 +269,24 @@ def test_drop_non_empty_list(self, index, drop_labels):
         with pytest.raises(KeyError, match="not found in axis"):
             DataFrame(index=index).drop(drop_labels)
 
+    @pytest.mark.parametrize("empty_listlike", [[], DatetimeIndex([])])
+    def test_drop_empty_listlike_non_unique_datetime_index(self, empty_listlike):
+        # GH#27994
+        data = {
+            "column_a": [5, 10],
+            "column_b": ["one", "two"]
+        }
+        index = [
+            Timestamp("2021-01-01"),
+            Timestamp("2021-01-01")
+        ]
+        df = DataFrame(data, index=index)
+
+        # Passing empty list-like should return the same DataFrame.
+        expected = df.copy()
+        dropped = df.drop(empty_listlike)
```

Comment:
Please call result. missed that previously

Reply:
In doc/source/development/contributing_codebase.rst the example they use is:

`def test_pivot(self):
        data = {
            'index' : ['A', 'B', 'C', 'C', 'B', 'A'],
            'columns' : ['One', 'One', 'One', 'Two', 'Two', 'Two'],
            'values' : [1., 2., 3., 3., 2., 1.]
        }

        frame = DataFrame(data)
        pivoted = frame.pivot(index='index', columns='columns', values='values')

        expected = DataFrame({
            'One' : {'A' : 1., 'B' : 2., 'C' : 3.},
            'Two' : {'A' : 1., 'B' : 2., 'C' : 3.}
        })

        assert_frame_equal(pivoted, expected)`

I was trying to use similar naming but 'result' makes sense as well and I can change this.

Reply:
Hm thanks for linking the contrib docs, should think about changing them. We use result most of the time